packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
+
+export interface TabsItemInfo extends IPressableState {
+  /**
+   * Disables the TabsItem.
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * TabsItem icon.
+   */
+  icon?: boolean;
+
+  /**
+   * TabsItem text.
+   */
+  headerText?: boolean;
+
+  /**
+   * Indicates if TabsItem is selected.
+   */
+  selected: boolean;
+
+  /**
+   * Key for tabItem
+   */
+  key: string;
+}
+
+export interface TabsItemState {
+  info: TabsItemInfo;
+}
+
+// Props for the tabs item
+export interface TabsItemProps extends IButtonProps {
+  /*
```

Comment:
nit: I think the commenting syntax for JSDoc is supposed to be two asterisks following the opening slash (so similar to the comments in the Info interface)